tests/FSharpx.Tests/EnumTest.fs

```diff
@@ -23,4 +23,28 @@ let ``parse throws an exception when it fails to parse`` ()=
     let parseEnglish ()=
         let x = Enum.parse("English") : LanguageOptions
         ()
-    Assert.Throws<System.Exception>( TestDelegate( parseEnglish ) )  
\ No newline at end of file
+    Assert.Throws<System.Exception>( TestDelegate( parseEnglish ) )  
+
+[<Test>] 
+let ``isDefined returns true when valid simple enum value is checked`` ()=
+    Assert.IsTrue(Enum.isDefined LanguageOptions.CSharp && Enum.isDefined LanguageOptions.FSharp && Enum.isDefined LanguageOptions.VB)
+
+[<Test>] 
+let ``isDefined returns false when not valid simple enum value is checked`` ()=
+    let invalidEnum : LanguageOptions = enum 3
+    Assert.IsFalse(Enum.isDefined invalidEnum)
+
+[<System.Flags>]
+type FlaggedLanguageOptions =
+    | FSharp = 0
+    | CSharp = 1
+    | VB = 2
+
+[<Test>] 
+let ``isDefined returns true when valid flagged enum value is checked`` ()=
+    Assert.IsTrue(Enum.isDefined (FlaggedLanguageOptions.CSharp ||| FlaggedLanguageOptions.FSharp ||| FlaggedLanguageOptions.VB))
```

Comment:
Minor:
```fsharp
let validFlaggedEnum = FlaggedLanguageOptions.CSharp ||| FlaggedLanguageOptions.FSharp ||| FlaggedLanguageOptions.VB
Assert.IsTrue (Enum.isDefined validFlaggedEnum)
```
would IMO make the intention even more explicit and the test more readable.

Reply:
I like this change :)